src/common/popup/components/error-details.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import StackTrace from 'stacktrace-js';
+
+import CopyButton from './copy-button';
+
+import ErrorShape from '../utils/error-shape';
+
+class ErrorDetails extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stackframes: null,
+    };
+  }
+
+  componentDidMount() {
+    const { error } = this.props;
+
+    StackTrace.fromError(error)
+      .then(stackframes => this.setState({ stackframes }))
+      .catch(err => console.error(err));
+  }
+
+  render() {
+    const { error } = this.props;
+    const { stackframes } = this.state;
+
+    if (!stackframes) return null;
+
+    const stack = stackframes.map(sf => sf.toString()).join('\n');
+    const info = [error.message, stack].join('\n\n');
+
+    return (
+      <div className="p-2 text-left">
+        <p className="font-weight-bold">{error.message}</p>
+        <pre className="overflow-auto">
+          <code>{stack}</code>
+        </pre>
+        <CopyButton value={info}>Copy error details</CopyButton>
```

Comment:
It is so fun that we could reuse this one :D

Reply:
What do you mean?

Reply:
I mean it is fun that we already have a copy button around as a default feature of tickety tick. In other projects one would have needed to build this stuff :D.  
No action required.

Reply:
Haha, I see. Yep 😁